bundles/org.openhab.binding.tesla/README.md

```diff
@@ -299,19 +299,19 @@ sitemap main label="Main"
         Frame
         {
             Switch item=TeslaAutoconditioning label="Enable Heat or AC"
-            Setpoint item=TeslaTemperature step=0.5 minValue=65 maxValue=78 label="Auto Conditioning Temperature [%.1f °F]"
-            Text item=TeslaInsideTemperature label="Inside Temperature [%.1f °F]" valuecolor=[<=32="blue",>95="red"]
-            Text item=TeslaOutsideTemperature label="Outside Temperature [%.1f °F]" valuecolor=[<=32="blue",>95="red"]
+            Setpoint item=TeslaTemperature step=0.5 minValue=65 maxValue=78 label="Auto Conditioning Temperature [%.1f %unit%]"
+            Text item=TeslaInsideTemperature label="Inside Temperature [%.1f %unit%]" valuecolor=[<=32="blue",>95="red"]
+            Text item=TeslaOutsideTemperature label="Outside Temperature [%.1f %unit%]" valuecolor=[<=32="blue",>95="red"]
         }
         Frame
         {
             Text item=TeslaBatteryLevel
             Text item=TeslaUsableBatteryLevel
             Text item=TeslaPower
             Text item=TeslaBatteryCurrent label="Current [%.1f]"
-            Text item=TeslaBatteryRange label="Battery Range [%.1f miles]"
-            Text item=TeslaEstBatteryRange label="Battery Est Range [%.1f miles]"
-            Text item=TeslaIdealBatteryRange label="Battery Ideal Range [%.1f miles]"
+            Text item=TeslaBatteryRange label="Battery Range [%.1f %unit%]"
```

Comment:
In the .items example there is no dimension for this item, so this will not work:
```java
Number TeslaBatteryRange {channel="account:model3:myaccount:mycar:batteryrange"}
```

However, I can see that indeed the channel is defined as `Number:Length`, so if you change the example to:
```
Number:Length TeslaBatteryRange {channel="account:model3:myaccount:mycar:batteryrange"}
```

it will work.

Or even better (since default units like "m" is not ideal for such long distance):
```
Number:Length TeslaBatteryRange {channel="account:model3:myaccount:mycar:batteryrange", unit="km"}
```

The same applies to the next two items.

Reply:
Thanks! You still need to add the dimension: `Number:Length` 😉